admin/bin/check-ps-report-for-corrupt-data.js

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env node
+'use strict'
+const fs = require('fs')
+const csv = require('fast-csv')
+const startTime = Date.now()
+const R = require('ramda')
+const RA = require('ramda-adjunct')
+
+// const timeRegex =  /^\d{1,2}:\d{2}:\d{2} (AM|PM)$/i // Not quite right - PM instead pm and single digit hours
+const timeRegex = /^\d{2}:\d{2}:\d{2} (AM|PM)$/ // Correct
+
+// Check for corrupt data in the PS report
+function help () {
+  console.log(`usage: ${process.argv[0]} <ps-file-to-inspect>`)
+}
+
+function report (prop, row, val) {
+  // console.log()
+  console.log(`${prop} field not matched for ${row['AttemptId']} val was '${val}'`)
+}
+
+function reportMissing (row, prop) {
+  process.stderr.write(`${prop} field missing for ${row['AttemptId']}\n`)
+}
+
+function reportProgress (count) {
+  // process.stdout.write('.')
```

Comment:
required?

Reply:
if it's potentially useful then keep it.  Just pointing out in case you forgot about some dead code 👍